tests/security/apparmor_profile/usr_sbin_smbd.pm

```diff
@@ -36,18 +36,26 @@ sub samba_server_setup {
     y2_module_guitest::launch_yast2_module_x11(module => "samba-server", target_match => "samba-server-installation", match_timeout => 200);
 
     send_key "alt-w";
+    send_key "ctrl-a";
+    send_key "delete";
     type_string("WORKGROUP");
     send_key_until_needlematch("samba-server-configuration", 'alt-n', 10, 2);
     send_key "alt-s";
     assert_screen("samba-server-configuration-shares");
     send_key "alt-a";
     assert_screen("samba-server-configuration-shares-newshare");
     send_key "alt-n";
+    send_key "ctrl-a";
```

Comment:
Maybe you can add wait_screen_still, then do not need to 'ctrl-a' and 'delete' any more.

Reply:
> Maybe you can add wait_screen_still, then do not need to 'ctrl-a' and 'delete' any more.

It does not work unfortunately.